main.py

```diff
@@ -94,11 +95,14 @@ def searchProgram():
 
 
 def init_logger():
-    logFileName = os.path.join("logs", str(datetime.today().date()) + "_" +
-                                       str(datetime.today().hour) + "." +
-                                       str(datetime.today().minute) + "." +
-                                       str(datetime.today().second) + ".log")
-    
+    baseDir = os.path.dirname(os.path.realpath(__file__))
+    logFileName = os.path.join(baseDir, "logs", str(datetime.today().date()) + "_" +
+                               str(datetime.today().hour) + "." +
+                               str(datetime.today().minute) + "." +
+                               str(datetime.today().second) + ".log")
+    with open(logFileName, 'w') as write_file:
```

Comment:
This isn't necessary, doing `logger.debug("main/init_logger: Logger Initialized")` already does this for you